Publish Release failing due to key errors.

Publishing the release workflow failing due to updates in version reference.

```
Run f = open('vintage-story-version.yaml', 'r') ; vs_version_yaml = yaml.safe_load(f) ; print(vs_version_yaml['vs_version']) ; f.close() >> $GITHUB_OUTPUT
/home/runner/work/_temp/0464a4cb-c3a0-4dee-af76-f0bffed6f49b.sh: line 1: syntax error near unexpected token `('
Error: Process completed with exit code 2.
```

--- a/.github/workflows/release-workflow.yaml
+++ b/.github/workflows/release-workflow.yaml
@@ -80,7 +80,7 @@ jobs:
         run: echo "repository=${{ github.repository }}" | tr '[:upper:]' '[:lower:]' >> $GITHUB_OUTPUT
 
       - id: VintageStoryVersion
-        run: f = open('vintage-story-version.yaml', 'r') ; vs_version_yaml = yaml.safe_load(f) ; print(vs_version_yaml['vs_version']) ; f.close() >> $GITHUB_OUTPUT
+        run: f = open('vintage-story-version.yaml', 'r') ; vs_version_yaml = yaml.safe_load(f) ; print(vs_version_yaml['vs_stable_version']) ; f.close() >> $GITHUB_OUTPUT
 
       - name: Build and push Docker images
         id: push

